ext/opentelemetry-ext-http-requests/tests/test_requests_integration.py

```diff
@@ -81,18 +110,18 @@ def test_invalid_url(self):
             {"component": "http", "http.method": "POST", "http.url": url},
         )
 
-    def test_disable(self):
-        opentelemetry.ext.http_requests.disable()
+    def test_uninstrument(self):
+        http_requests.RequestsInstrumentor().uninstrument()
         result = requests.get(self.URL)
         self.assertEqual(result.text, "Hello!")
         span_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(span_list), 0)
+        # instrument again to avoid annoying warning message
+        http_requests.RequestsInstrumentor().instrument()
```

Comment:
should this be a try/ finally clause? so that in the case that this test fails, the instrumentation is turned back on for other tests?

Reply:
ah ok, looks like this is handled in the setup/teardodwn anyway.